src/App.vue

```diff
@@ -1,19 +1,25 @@
 <template>
   <v-app>
     <v-content>
-      <Home />
+      <Home v-bind:getTaskPath="getTaskPath" />
     </v-content>
   </v-app>
 </template>
 
 <script>
+import Home from "./components/Home";
+import config from '../public/config'
 
-import Home from './components/Home'
 export default {
+  data() { return {
+      getTaskPath : "",
+    }
+  },
   components: {
     Home: Home
   },
-  created () {
+  created() {
+    this.getTaskPath = config.getTaskPath;
```

Comment:
I think you could initialize the value of `getTaskPath` in  the `data()` function to `config.getTaskPath` to avoid using a `created()` function here.